### DISCUSSION TITLE
For which languages is syntax highlighting available?

### DISCUSSION THREAD

**Discussion by highball-it** [requarks/wiki#4762] [Help / Questions] [answered]
Hi,
I've read the docs about syntax highlighting and found the list of supported languages. I'm wondering how I could check if one language should be available. Because I can't get syntax highlighting to work for Caché Object Script (cos, cos)

```
```cos
Class Foo.Bar
{
}
```

Thank you!

**Comment by NGPixel** [comment, marked as answer]
https://prismjs.com/#supported-languages

**Reply by highball-it** [reply]
Thank you!

In the docs under https://docs.requarks.io/en/editors/markdown _Code blocks_ it references the following highlight.ja docs https://github.com/highlightjs/highlight.js#supported-languages
